lib/nopt-lib.js

```diff
@@ -17,11 +17,22 @@ const getType = (k, { types, fallbackTypes }) => {
   return [hasType, type]
 }
 
-function nopt (args, { types, fallbackTypes, shorthands, typeDefs, invalidHandler, typeDefault }) {
+const isTypeDef = (type, def) => def && type === def
+const hasTypeDef = (type, def) => def && type.indexOf(def) !== -1
+const doesNotHaveTypeDef = (type, def) => def && type.indexOf(def) === -1
```

Comment:
suggestion: `!hasTypeDef(type, def)` is something we could do.

Reply:
:thumbsup: